src/main/kotlin/Figure.kt

```diff
@@ -0,0 +1,13 @@
+class Figure(col: String) {
```

Comment:
Not the best choice for constructor type. I think, there should be a way to pass `Color` here.